Products/CMFPlone/tests/emaillogin.txt

```diff
@@ -172,20 +179,19 @@ The current email address of course works fine for logging in:
 
 Picking the e-mail address of another user should of course fail:
 
-    >>> browser.open('http://nohost/plone/@@personal-information')
-    >>> browser.getControl('E-mail').value = 'username@example.org'
-    >>> browser.getControl('Save').click()
-    >>> self.assertFalse('Changes saved.' in browser.contents)
-    >>> browser.open('http://nohost/plone/logout')
-
+A    >>> browser.open('http://nohost/plone/@@personal-information')
```

Comment:
@tisto , are you talking about this test? These lines that we're commenting on seem to work now: https://github.com/plone/Products.CMFPlone/commit/5976ace5c77acbf5ec356085001cd4f24be97f11